apps/fluent-tester/src/FluentTester/testPages.windows.ts

```diff
@@ -62,6 +63,11 @@ export const tests = [
     component: CheckboxTest,
     testPage: HOMEPAGE_CHECKBOX_BUTTON,
   },
+  {
```

Comment:
We only added that control to win32 as we only spec'd it out for win32, windows and macos. Even then, there are style, keyboarding, and accessibility differences among these platforms. 

We added it to the windows tester app even though it is not currently working because of a dependency to FocusZone, like RadioGroup.